tests/ttnn/integration_tests/bert_tiny/test_bert_tiny.py

```diff
@@ -0,0 +1,235 @@
+    reset_seeds,
+):
+    model_name = str(model_location_generator("mrm8488/bert-tiny-finetuned-squadv2", model_subdir="Bert"))
+    hugging_face_reference_model = BertForQuestionAnswering.from_pretrained(model_name, torchscript=False)
+
+    encoder_idx = 0
+    pytorch_attention_model = hugging_face_reference_model.bert.encoder.layer[encoder_idx].attention
+    config = hugging_face_reference_model.config
+
+    parameters = preprocess_model_parameters(
+        initialize_model=lambda: pytorch_attention_model,
+        device=device,
+        convert_to_ttnn=lambda *_: True,
+    )
+
+    input = (torch.rand(batch_size, 1, 128, hugging_face_reference_model.config.hidden_size) * 2) - 1
+    torch_attention_mask = torch.zeros(1, 128)
+    pytorch_out = pytorch_attention_model(input.squeeze(1), attention_mask=torch_attention_mask)[0]
+
+    tt_input = ttnn.from_torch(input.squeeze(1), device=device, dtype=ttnn.bfloat16, layout=ttnn.TILE_LAYOUT)
+    tt_attention_mask = ttnn.from_torch(
+        torch_attention_mask, device=device, dtype=ttnn.bfloat16, layout=ttnn.TILE_LAYOUT
+    )
+
+    tt_output = bert_attention(
+        config=config,
+        hidden_states=tt_input,
+        attention_mask=tt_attention_mask,
+        parameters=parameters,
+        device=device,
+    )
+    tt_output = ttnn.from_device(tt_output)
+    tt_output = ttnn.to_torch(tt_output)
+
+    assert_with_pcc(pytorch_out, tt_output, 0.99)
+
+
+@pytest.mark.parametrize("device_params", [{"l1_small_size": 24576}], indirect=True)
+@pytest.mark.parametrize(
+    "batch_size",
+    ((8),),
+)
+def test_bert_intermediate_inference(
+    batch_size,
+    model_location_generator,
+    device,
+    reset_seeds,
+):
+    model_name = str(model_location_generator("mrm8488/bert-tiny-finetuned-squadv2", model_subdir="Bert"))
+    hugging_face_reference_model = BertForQuestionAnswering.from_pretrained(model_name, torchscript=False)
+
+    encoder_idx = 0
+    pytorch_intermediate_model = hugging_face_reference_model.bert.encoder.layer[encoder_idx].intermediate
+
+    parameters = preprocess_model_parameters(
+        initialize_model=lambda: pytorch_intermediate_model,
+        device=device,
+        convert_to_ttnn=lambda *_: True,
+    )
+
+    input = (torch.rand(batch_size, 1, 128, hugging_face_reference_model.config.hidden_size) * 2) - 1
+    pytorch_out = pytorch_intermediate_model(input).squeeze(0)
+
+    tt_input = ttnn.from_torch(input.squeeze(1), dtype=ttnn.bfloat16, layout=ttnn.TILE_LAYOUT, device=device)
+
+    tt_output = bert_intermediate(
+        hidden_states=tt_input,
+        parameters=parameters,
+        device=device,
+    )
+    tt_output = ttnn.from_device(tt_output)
+    tt_output = ttnn.to_torch(tt_output)
+    assert_with_pcc(pytorch_out.squeeze(1), tt_output, 0.99)
+
+
+@pytest.mark.parametrize("device_params", [{"l1_small_size": 24576}], indirect=True)
+@pytest.mark.parametrize(
+    "batch_size",
+    ((8),),
+)
+def test_bert_output_inference(
```

Comment:
does this test run anywhere?

Reply:
This is a sub-module test for Bert-Tiny model and this is tested in CIs.

Reply:
I see test_performance is calling test_bert_tiny.py::test_bert_for_question_answering but I cannot find any where this is called. Maybe I am missing it? Where is call for this test?

Reply:
test_bert_tiny.py::test_bert_for_question_answering is present in the below mentioned path [tests/ttnn/integration_tests/bert_tiny/test_bert_tiny.py](https://github.com/tenstorrent/tt-metal/compare/main...vignesh/ttnn_bert_tiny#diff-0a8c71d42b9f83650b262e8804fa85466fda47861d90a4e841eaaa7ab04f7bfbR192)

Reply:
I am asking which function is calling `test_bert_output_inference`. Does this test run anywhere?

Reply:
No, this isn't running anywhere; it's simply a test to test that specific sub-module.